packages/react-sdk/src/components/link-block.ws.tsx

```diff
@@ -0,0 +1,26 @@
+import { BoxLinkIcon } from "@webstudio-is/icons";
+import type { WsComponentMeta, WsComponentPropsMeta } from "./component-type";
+import { props } from "./__generated__/link-block.props";
+
+const presetStyle = {
+  boxSizing: {
+    type: "keyword",
+    value: "border-box",
+  },
+  display: {
+    type: "keyword",
+    value: "inline-block",
```

Comment:
Should it be inline block?

Reply:
yeah, its more likely use case, also wf uses it